scripts/import/import_all_sv_data.pl

```diff
@@ -1847,7 +1898,7 @@ sub cleanup {
   }
   
   # Column "tmp_clinic_name" in structural_variation
-  my $sth2 = $dbVar->prepare(qq{ SELECT count(*) FROM $sv_table WHERE clinical_significance is NULL AND $tmp_sv_clin_col is not NULL});
+  my $sth2 = $dbVar->prepare(qq{ SELECT count(*) FROM $sv_table WHERE (clinical_significance is NULL or clinical_significance = '') AND $tmp_sv_clin_col is not NULL});
```

Comment:
or should be upper case to be consistent. => 'NULL OR'

Reply:
It should. Thanks.